### PR TITLE
Add absorption chiller

### PR DESCRIPTION
**Pull request by zolanaj**

### CODE REVIEW

**Comment by zolanaj** [comment]
@NLaws I'll let you know when tests are passing.

**Comment by zolanaj** [comment]
@NLaws Xpress tests are passing, but the Cbc tests currently fail due to an error initializing Cbc:
`
┌ Warning: Could not use exact versions of packages in manifest, re-resolving
└ @ TestEnv ~/.julia/packages/TestEnv/S9yOn/src/activate_set.jl:62
ERROR: LoadError: InitError: could not load library "/home/runner/.julia/artifacts/e[5](https://github.com/NREL/REopt.jl/runs/6160843005?check_suite_focus=true#step:5:5)4c[6](https://github.com/NREL/REopt.jl/runs/6160843005?check_suite_focus=true#step:5:6)1de2448[7](https://github.com/NREL/REopt.jl/runs/6160843005?check_suite_focus=true#step:5:7)15375d627cc2deffc91d4a7815b/lib/libClp.so"
libdmumps.so: cannot open shared object file: No such file or directory 
`

Have you encountered this before?  Would updating Project.toml and/or Manifest.toml take care of this?

**Comment by NLaws** [comment]
> Have you encountered this before? Would updating Project.toml and/or Manifest.toml take care of this?

I have not. I am trying an update of the test env: https://github.com/NREL/REopt.jl/actions/runs/2222658143

File: src/core/reopt_inputs.jl [hunk 1]
```diff
@@ -495,6 +506,22 @@ function setup_existing_chiller_inputs(s::AbstractScenario, max_sizes, min_sizes
     return nothing
 end
 
+
+function setup_absorption_chiller_inputs(s::AbstractScenario, max_sizes, min_sizes, cap_cost_slope, 
+    segmented_techs, n_segs_by_tech, seg_min_size, seg_max_size, seg_yint, cop, thermal_cop, om_cost_per_kw
+    )
+    max_sizes["AbsorptionChiller"] = s.absorption_chiller.max_kw
+    min_sizes["AbsorptionChiller"] = s.absorption_chiller.min_kw
+    update_cost_curve!(s.absorption_chiller, "AbsorptionChiller", s.financial,
```
Comment (NLaws) [inline src/core/reopt_inputs.jl:515]: @zolanaj  does the AbsorptionChiller ever have a segmented cost curve? If not we can use the `effective_cost` function like [this](https://github.com/NREL/REopt.jl/blob/c9c0c474ab46bc38884aa970098eba4ff894fd49/src/core/reopt_inputs.jl#L502) to account for MACRS and remove the `AbsorptionChiller` fields that are always zero.

Comment (zolanaj) [reply src/core/reopt_inputs.jl]: > @zolanaj does the AbsorptionChiller ever have a segmented cost curve? If not we can use the `effective_cost` function like [this](https://github.com/NREL/REopt.jl/blob/c9c0c474ab46bc38884aa970098eba4ff894fd49/src/core/reopt_inputs.jl#L502) to account for MACRS and remove the `AbsorptionChiller` fields that are always zero.

@NLaws thanks for catching! no segmented cost curve exists to my knowledge.  I updated the cost curve in 63ad2df279e03a7bb210abd7265c428204df662b and will merge once I've confirmed that tests pass.

Comment (NLaws) [reply src/core/reopt_inputs.jl]: Thanks @zolanaj ! Can you please also remove the unused fields from the `AbsorptionChiller` struct?

Comment (zolanaj) [reply src/core/reopt_inputs.jl]: Thanks for catching @NLaws!  Removed in 45f90074701069dc684d6bcf274c3d40a167ffcd.

**Review by NLaws — APPROVED** [review]

**Pull request merged**